source/plugin/text/templates.rst

```diff
@@ -0,0 +1,141 @@
+==========================
+Working with TextTemplates
+==========================
+
+TextTemplate's are an easy and convenient way to store messages with variable elements. For instance, you may find
```

Comment:
No apostrophe needed, and this should also be in an inline code block with double back-ticks (``). Also, to append the 's', you should escape it with `\ s`